src/pages/[...slug].tsx

```diff
@@ -181,7 +182,13 @@ ContentPage.getLayout = (page: ReactElement) => {
 
   return (
     <RootLayout {...rootLayoutProps}>
-      <Layout {...layoutProps}>{page}</Layout>
+      <Layout {...layoutProps}>
+        <PageMetadata
```

Comment:
`image`, `canonicalUrl`, `author` are not being used here so they'll be always `undefined`, do we need these props then?

Reply:
I just looked through all uses of `PageMetadata` in the Gatsby repo, and we do use `image` in a couple of the React pages, and `canonicalUrl` is only used in the `tutorial` "template" file.

I don't see any usage of `author` in that repo, but I had added the optional Tutorial frontmatter `author` as a prop inside `[...tutorial].tsx`

So if we want to undo passing the `author` in the tutorial when it is available, we could ditch that one... the other two as they stand are still being used in other areas.

Reply:
On that note, noticing that in Gatsby repo we used `sourceUrl` from the tutorial frontmatter as the canonicalUrl here and I didn't include that... I'll add a patch to include it inside `[...tutorial].tsx`